src/platforms/javascript/common/configuration/transports.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Transports
+sidebar_order: 30
+description: "Transports let you change the way in which events are delivered to Sentry."
+notSupported:
+  - javascript.capacitor
+  - javascript.cordova
+  - javascript.electron
+  - javascript.wasm
+---
+
+The JavaScript SDK uses a `transport` to send events to Sentry. On modern
+browsers, the default transport uses the browsers `fetch` API to send events.
+
+The default transports drop events if they fail to send due to a lack of connection.
```

Comment:
```suggestion

The JavaScript SDK uses a `transport` to send events to Sentry. On modern browsers, most transports use the browsers' `fetch` API to send events. Transports will drop an event if it fails to send due to a lack of connection.
```
Let me know if this changes the meaning too much. If it's important to differentiate between a transport and a "default" transport, it might be good to explain the difference a bit more :)